android/java/res/layout/brave_talk_opt_in_layout.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 The Brave Authors. All rights reserved.
+     Use of this source code is governed by a BSD-style license that can be
+     found in the LICENSE file. -->
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="320dp"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/background_bat"
+        android:scaleType="center"
+        android:layout_alignParentTop="true"
+        android:layout_alignParentLeft="true"
+        android:contentDescription="@null" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:layout_alignParentTop="true"
+        android:paddingHorizontal="25dp"
+        android:paddingVertical="10dp">
+
+        <ImageView android:id="@+id/brave_talk_opt_in_image"
+            android:layout_width="match_parent"
+            android:layout_height="0px"
+            android:layout_weight="2"
+            android:src="@drawable/cashback"
```

Comment:
Once we include vector drawables, this should be replaced with app:srcCompat

Reply:
Unfortunately, Android Studio cannot import shadow for cashback icon. So resulting icon looks like this:
![wrong_icon](https://user-images.githubusercontent.com/86598290/133113024-0c27def5-4124-4150-a2d2-2cf63c75cb9e.png)

As I think there are two possible solutions:
1) Ask designer to remove shadow from icon and add borders.
2) Draw png for this cashback icon.

@deeppandya, Could you suggest me what to do in this case, please?
Other icons are drown using vector drawables.

Reply:
@aseren i think @codybanxs can help here. i would recomend to use vector drawables as we have size constraint in apk.

Reply:
I have found similar resource ic_onboarding_graphic_cashback.
Added using of this resource